src/LinearSolvers/RichardsonLinearSolvers.jl

```diff
@@ -0,0 +1,111 @@
+"""
+    struct RichardsonLinearSolver <: LinearSolver 
+      ...
+    end
+
+    RichardsonLinearSolver(ω,IterMax;Pl=nothing,rtol=1e-10,atol=1e-6,verbose=true,name = "RichardsonLinearSolver")
+
+  Richardson Iteration, with an optional left preconditioners `Pl`.
+
+  The relaxation parameter (ω) can either be of type Float64 or Vector{Float64}. 
+  This gives flexiblity in relaxation.  
+"""
+
+struct RichardsonLinearSolver<:Gridap.Algebra.LinearSolver
+    ω::Union{Vector{Float64},Float64}
+    Pl::Union{Gridap.Algebra.LinearSolver,Nothing}
+    IterMax::Int64
+    log::ConvergenceLog{Float64}
+end
+
+function RichardsonLinearSolver(ω,IterMax;Pl=nothing,rtol=1e-10,atol=1e-6,verbose=true,name = "RichardsonLinearSolver")
```

Comment:
Any reason we are not using a kwarg `maxiter` like most of the other solvers? I think it might make it more consistent.

Reply:
I changed it to `maxiter` to maintain uniformity.